contact-us.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact Us</title>
+</head>
+
+<body>
+    <section>
+        <div class="container">
+            <div class="contactinfo">
+                <div>
+                    <h2>Contact Info</h2>
+                    <ul class="info">
+                        <li>
+                            <span><img src="https://cdn.picpng.com/pin/pin-location-map-icon-53136.png" alt=""></span>
+                            <span>Maharashtra, Navi Mumbai <br>
```

Comment:
First sector 3C, Khargar, Navi Mumbai, Maharashtra, then pincode. This is the general format of writing address. This  appears pleasant to the users.

Reply:
General Format of writing address has been applied!